browser/ipfs/sources.gni

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+import("//brave/components/ipfs/buildflags/buildflags.gni")
+import("//extensions/buildflags/buildflags.gni")
+
+brave_browser_ipfs_sources = []
+
+brave_browser_ipfs_deps = [ "//brave/components/ipfs/buildflags" ]
+
+if (ipfs_enabled) {
+  brave_browser_ipfs_sources += [
+    "//brave/browser/ipfs/content_browser_client_helper.cc",
+    "//brave/browser/ipfs/content_browser_client_helper.h",
+    "//brave/browser/ipfs/import/ipfs_import_controller.cc",
+    "//brave/browser/ipfs/import/ipfs_import_controller.h",
+    "//brave/browser/ipfs/import/save_package_observer.cc",
+    "//brave/browser/ipfs/import/save_package_observer.h",
+    "//brave/browser/ipfs/ipfs_host_resolver.cc",
+    "//brave/browser/ipfs/ipfs_host_resolver.h",
+    "//brave/browser/ipfs/ipfs_service_factory.cc",
+    "//brave/browser/ipfs/ipfs_service_factory.h",
+    "//brave/browser/ipfs/ipfs_tab_helper.cc",
+    "//brave/browser/ipfs/ipfs_tab_helper.h",
+  ]
+
+  brave_browser_ipfs_deps += [
+    "//brave/components/ipfs",
+    "//brave/components/services/ipfs/public/mojom",
+    "//components/user_prefs",
+  ]
```

Comment:
The sources depend on more targets that the three ones listed here, see what I came up with in https://github.com/brave/brave-core/pull/8538/commits/8515943823d236963abf3d90af634ff0da578eb3#diff-5a1def418fe4a60c244b8b3e9f355aebf361c47a4d811539c324de5972648f7d for reference:
```
  brave_browser_ipfs_deps += [
    "//brave/browser/profiles:util",
    "//brave/common",
    "//brave/components/ipfs",
    "//brave/components/services/ipfs/public/mojom",
    "//chrome/browser:browser_process",
    "//chrome/common:channel_info",
    "//chrome/common:constants",
    "//components/keyed_service/content",
    "//components/prefs",
    "//components/user_prefs",
    "//extensions/buildflags",
  ]
```